indigo/commands.sbt

```diff
@@ -208,26 +207,20 @@ addCommandAlias(
 
 addCommandAlias(
   "crossBuildIndigo",
-  applyCrossCommand(coreProjects, "compile")
+  applyCrossCommand(allProjects, "compile")
 )
 addCommandAlias(
   "crossLocalPublishIndigo",
-  applyCrossCommand(
-    coreProjects.filterNot(name => name == "sandbox" || name == "perf"),
-    "publishLocal"
-  )
+  applyCrossCommand(allProjects, "publishLocal")
 )
 addCommandAlias(
   "crossLocalPublishNoClean",
-  List(
-    "crossBuildIndigo",
```

Comment:
Removed it, because looking at the logs I think the `publish` task depends on the `compile` task